_layouts/post.html

```diff
@@ -79,6 +79,11 @@ <h1 class="title">{{ page.title }}</h1>
         <i class="fa fa-{{ social.icon | remove_first: '-square' }}-square fa-lg"></i>
       </a>
     {% endif %}
+	{% if social.name == "Reddit" and social.share == true %}
+		<a href="//www.reddit.com/submit" onclick="window.location = '//www.reddit.com/submit?url=' + encodeURIComponent('{{ full_url }}') + '&title={{page.title}}'; return false">
+			<i class="fa fa-{{ social.icon | remove_first: '-square' }}-square fa-lg"></i>
+		</a>
+	{% endif %}
```

Comment:
The most nit-picky thing: the indentation doesn't line up. Do you have an EditorConfig plugin for your IDE/text editor?